frontend/spec/features/product_page_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
```

Comment:
@tomurb we don't need this spec, there's a lot of other specs in spree frontend gem for this action